README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Overview
 
-The Prometheus Operator provides [Kubernetes](https://kubernetes.io/) native deployment and management of
+The Prometheus Operator provides [Kubernetes](https://kubernete.io/) native deployment and management of
```

Comment:
This is added to check if action works if doc check fails, will remove once confirmed working